assets/pages/bundles-landing/support-landing-ab-test/supportLanding.scss

```diff
@@ -291,25 +291,38 @@
     }
 
     .subscription-bundle__price {
-      font-family: $gu-text-sans-web;
+      font-family: $gu-egyptian-web;
       font-size: 24px;
       line-height: 28px;
       padding: $gu-v-spacing 0;
       color: gu-colour(neutral-1);
+      font-weight: 500;
     }
 
     .subscription-bundle__price-amount {
       font-size: 36px;
       line-height: 28px;
-      font-weight: bold;
     }
 
     .subscription-bundle__copy {
-      font-family: $gu-text-egyptian-web;
-      font-size: 18px;
-      line-height: 24px;
       color: gu-colour(neutral-2);
       flex-grow: 1;
+
+      .component-feature-list {
+        list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 28 28"><style>.st0{fill:#333;}</style><path class="st0" d="M7.885 25.74L0 14.82l3.22-3.217L8.3 16.74 24.695 2.26 28 5.565"/></svg>');
```

Comment:
Is there a way to avoid this hardcoded SVG?

Reply:
This is a problem I hit first time round with the bundles landing page (this is the same solution). It's difficult to do it in a way that works cross-browser. Last time I ended up with alignment issues all over the place depending on the browser you open the page with (and the SVG doesn't show at all in IE10).

It's possible that Amo is going to give me a new SVG for this, and if he does I'll see if I can come up with another solution, but I'm not 100% confident 😐.